src/main/java/org/example/servletstexts/TextsServlet.java

```diff
@@ -94,19 +72,60 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
 
             if (text != null && !text.trim().isEmpty()) {
                 texts.put(currentId, text);
+
                 ObjectNode responseBody = objectMapper.createObjectNode();
                 responseBody.put("text", text);
                 responseBody.put("message", "Text saved with id: " + currentId);
                 responseBody.put("textId", currentId);
                 respWriter.write(responseBody.toString());
+
                 currentId++;
             } else {
                 resp.setStatus(HttpServletResponse.SC_BAD_REQUEST);
                 respWriter.write("Text cannot be empty");
             }
         } catch (IOException e) {
             resp.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
-            resp.getWriter().write("Internal server error");
+            resp.getWriter().write(MESSAGE_INTERNAL_SERVER_ERROR);
+        }
+    }
+
+    private void responseProcessForDoGetMethod(String idString, PrintWriter respWriter, HttpServletResponse response) {
```

Comment:
`responseProcessForDoGetMethod` - strange words order 
` String idString` - why type iswritten in variable name ? `String id` it is quite obvious that 'id' is type of string and no reason to mention this fact in name.
`PrintWriter respWriter` - Can't you get it from response ?